src/content/en/fundamentals/primers/service-workers/what-do-they-do.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: shared/narrow
+published: false
+title: "What do they do?"
+description: "Site search is vital for helping mobile users find what they’re looking for in a hurry"
+authors:
+- josephmedley
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 2
+---
+
+<p class="intro">
+  Service workers provide web pages with capabilities that make them more like 
+  native apps. Here are a list of characteristics and apps or sites that 
+  demonstrate their use.
+</p>
+
+### Responsiveness
+
+What if you could build a web page that took voice 
+recordings? Google's sample app called [Voice Memos](https://voice-memos.appspot.com/) 
+does exactly that. For a web-based memo app a slow network would mean slow 
+playback. Voice Memos solves this by caching memos on the device. The code is 
```

Comment:
My understanding of Voice Memos is that it uses IndexedDB, outside of the service worker, to store the voice memos. That's useful and valid, but it doesn't fit into the section's purpose, which is explaining what service workers do.